drivers/pwm/pwm_led_esp32.c

```diff
@@ -169,7 +169,18 @@ static int pwm_led_esp32_timer_config(struct pwm_ledc_esp32_channel_config *chan
 		return 0;
 	}
 
-	return -EINVAL;
+	/**
+	 * ESP32 - S2,S3 and C3 variants have only 14 bits counter.
+	 * where as the plain ESP32 variant has 20 bits counter.
+	 * application failed to set low frequency(1Hz) in S2, S3 and C3 variants.
+	 * to get very low frequencies on these variants,
+	 * frequency needs to be tuned with 18 bits clock divider.
+	 * so select the slow clock source (1MHz) with highest counter resolution.
+	 * this can be handled on the func 'pwm_led_esp32_timer_set' with 'prescaler'.
+	 */
+	channel->clock_src = LEDC_REF_TICK;
```

Comment:
This line is redundant.

Reply:
@LucasTambor thanks for the review comment.
I removed that duplicate line.